README.md

```diff
@@ -34,15 +39,25 @@ For `2.5.2`, any of the following directory names would work:
 Note that the `rbis` script will include _all_ matching directory's `.rbi` files.
 
 As an example, if the following directories existed:
- - `lib/json-schema/all`
- - `lib/json-schema/>1.0`
- - `lib/json-schema/~>2.5`
- - `lib/json-schema/>2.6`
+   - `lib/json-schema/all`
+   - `lib/json-schema/>1.0`
+   - `lib/json-schema/~>2.5`
+   - `lib/json-schema/>2.6`
 
 For `json-schema@2.5.2`, all of the `.rbi` files that exist in the following directories would be loaded:
- - `lib/json-schema/all`
- - `lib/json-schema/>1.0`
- - `lib/json-schema/~>2.5`
+   - `lib/json-schema/all`
+   - `lib/json-schema/>1.0`
+   - `lib/json-schema/~>2.5`
+
+### Writing RBIs
+
+When writing an RBI file for sorbet-typed, you can start with a file in a project of your own. For example, if you have a Rails application and want to improve the type signatures for ActiveRecord, you could go into `sorbet/sorbet-typed/activerecord/all/activerecord.rbi` in your repo, and then start editing from there. This is useful because it allows you to test changes alongside real Ruby code using the gem.
```

Comment:
You can also just add hand-written rbis in a separate directory of your own in your project, like `rbi/activerecord/all/activerecord.rbi`. Sorbet will pick this up too, and its contents won't get overwritten the next time you update the sorbet-typed rbis. Maybe we should encourage people to go that route?

Reply:
I think suggestion by @connorshea is better as it fosters sharing `rbi`s for gems